charts/schema-loading/templates/batchjob-schema-loading.yaml

```diff
@@ -55,6 +56,18 @@ spec:
         - "-j"
         - "{{ .Values.schemaLoading.contactPoints }}"
         {{- end }}
+        {{- else }}
+        - "--config"
+        - "database.properties"
+        {{- if eq .Values.schemaLoading.schemaType "ledger" }}
+        - "--coordinator"
+        {{- end }}
+        {{- end }}
+        {{- if .Values.schemaLoading.commandArgs }}
+        {{- range .Values.schemaLoading.commandArgs }}
+        - {{ . | quote }}
+        {{- end }}
+        {{- end }}
```

Comment:
If array is specified as a value of `schemaLoading.commandArgs`, it will be listed in the `spec.containers[].args`.